test-apps/basic/main.go

```diff
@@ -0,0 +1,30 @@
+package main
```

Comment:
We'll probably want the test apps to live in the `internal/` directory, like `internal/test-apps` to be clear that these aren't for public consumption :)